db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+    guild = models.ForeignKey(Guild, on_delete=models.CASCADE, null=True)
```

Comment:
I don't think `CASCADE` should be applied to this field. What happens if guild leader decides to delete the guild? Should all the players in it be deleted? :)

Reply:
Thank's for your review! I've done almost all changes, but I don't know how to make my code more efficient.

Reply:
The solution I mean is not an improvement of Python code. It's an improvement of database interactions.

If you store the output of `.create()` methods in variables and only execute `.get()` when the entry already exists, it will result in less database calls overall.

There's also a method for this: `.get_or_create()`.

Reply:
I've tried to refactor my code with .get_or_create() method. I hope I took your point and did it right.

Reply:
Absolutely, GJ! :)